src/core/services/managers/marker-manager.ts

```diff
@@ -73,6 +81,10 @@ export class MarkerManager {
       title: marker.title
```

Comment:
please handle the animation here, please handle also null/undefined cases here as above.